ts/components/session/conversation/SessionRecording.tsx

```diff
@@ -33,6 +33,24 @@ function getTimestamp(asInt = false) {
   return asInt ? Math.floor(timestamp) : timestamp;
 }
 
+export interface StyledFlexWrapperProps {
+  flexDirection: 'row' | 'column';
+  marginHorizontal: string;
+}
+
+/**
+ * Generic wrapper for quickly passing in theme constant values.
+ */
+export const StyledFlexWrapper = styled.div`
+  display: flex;
+  flex-direction: ${(props: StyledFlexWrapperProps) => props.flexDirection};
```

Comment:
rather than having to duplicate the type of props here,
you can do

```
styled.div<StyledFlexWrapperProps>`....`
```
and typescript will understand that props is of this type,
so you can only do 

```
  flex-direction: ${props => props.flexDirection};
```

Reply:
actually,
you only instantiate a single of those StyledFlexWrapper.

So no need for props at all, 
you can just do 

```
export const StyledFlexWrapper = styled.div`

[whatever you already have in it\

flex-direction:row;
 marginHorizontal: (props => props.theme.margins.xs);
>
`

```